### DISCUSSION TITLE
Breakpoints variables in CSS modules

### DISCUSSION THREAD

**Discussion by Tomycl** [mantinedev/mantine#5159] [Q&A] [answered]
I followed the section on [Breakpoints variables in CSS modules](https://mantine.dev/styles/responsive/) and configured Postcss to use the `Feature` component provided by `Mantine UI`, but my vscode still reported an error for statements like `$mantine-breakpoint-md`. How I can solve it?

**Comment by rtivital** [comment, marked as answer]
Install https://marketplace.visualstudio.com/items?itemName=csstools.postcss

**Reply by Tomycl** [reply]
Thank you. Problem solved!
